pipelines/fbc-builder/patch.yaml

```diff
@@ -1,4 +1,11 @@
 ---
+- op: add
+  path: /spec/description
+  value: |
+    This pipeline is ideal for building and verifying xref:/advanced-how-tos/building-olm.adoc#building-the-file-based-catalog[file-based catalogs].
+
+    _Uses `buildah` to create a container image. Its build-time tests are limited to verifying the included catalog and not to scanning the image itself.
```

Comment:
nit: the sentence `Its build-time tests are limited to verifying the included catalog and not to scanning the image itself.` sounds a bit unclear to me 

How about something like `Its build-time tests are limited to verifying the included catalog and do not scan the image.`?